source/whats-new.txt

```diff
@@ -12,3 +12,250 @@ What's New
+* :ref:`Version 1.0 <version-1.0>`
+
+
+.. _version-1.10.0:
+.. _version-1.10:
+
+What's New in 1.10
+--------------------
+
+.. important:: Upgrade to Version 1.10.1 or Higher 
+
+  The 1.10.1 Go driver patches a bug that can cause data corruption when 
+  rotating :ref:`Data Encryption Keys <csfle-key-architecture>` (DEKs) encrypted 
+  with a :ref:`Customer Master Key <csfle-key-architecture>` hosted on Google 
+  Cloud Key Management Service or Azure Key Vault.
+
+New features of the 1.10 Go driver release include: 
+
+- Added full support for MongoDB 6.0.
+  
+- Support for new features related to :ref:`qe-manual-feature-qe`,
+  including new options for automatic and manual encryption.
+
+- Support for the new Automatic Encryption Shared Library, which replaces the 
+  ``mongocryptd`` process. The shared library requires MongoDB v6.0 Enterprise 
+  or later and ``libmongocrypt`` 1.5.0 or later.
+
+- Added :ref:`clustered index <golang-clustered-indexes>` creation support.
+
+- Added a new API and ``ClientEncryption`` entity operations for encryption key 
+  management.
+
+- Added a ``Timeout`` client option to set a default context timeout for 
+  each operation sent through that client.
+
+- Implemented a patch to default data to either ``primitive.M`` or 
+  ``primitive.D`` when decoding empty types.
+
+- Support for encoding atypical map key types for data that can be unmarshalled 
+  into a textual representation of itself. 
+
+- Performance optimizations including:
+  
+  - Improved full document requests for before and after updates in change 
+    stream events.
+
+  - Improved PRN and UUID generation 
+
+  - Reduced memory consumption when compressing wire messages.
+
+- Added troubleshooting support for `frequently encountered issues 
+  <https://github.com/mongodb/mongo-go-driver/blob/master/docs/common-issues.md>`__
+
+
+.. _version-1.9:
+
+What's New in 1.9
+-------------------
+
+New features of the 1.9 Go driver release include: 
+
+- Improved connection storm mitigation.
+
+- Added ``Custom```` options to aggregate and change stream operations.
+
+- Added a ``Let`` option to most CRUD commands that specifies parameters for use 
+  in an aggregate expression context. ``Let`` must be a document that maps 
+  parameter names to values that are constant or closed expressions without 
+  references to document fields. MongoDB v5.0 or later is required.
+
+- Added new constructor functions that create ``Cursor`` and ``SingleResult`` 
+  instances from marshalable and non-nil BSON documents.
+
+
+.. _version-1.8:
+
+What's New in 1.8
+-----------------
+
+New features of the 1.8 Go driver release include: 
+
+- Added full support for MongoDB 5.1.
+
+- Support for KMIP (Key Management Interoperability Protocol) as a KMS 
+  provider for client side encryption. 
+
+- Redesigned driver connection pool for low operation ``Context`` timeouts and 
+  to reduced connection churn. Behavior changes include: 
```

Comment:
Q: Should this be "reduce" ? Or delete the word "to"? I'm not sure which one is correct.

Reply:
Typo on my end! Should be "reduce"